scripts/validator-info

```diff
@@ -794,19 +795,47 @@ def main():
     if not info_paths:
         print('There are no info files in {}'.format(args.basedir))
         return
-    for file_path in info_paths:
-        logger.info("Reading file {} ...".format(file_path))
-        if args.verbose:
+    if args.json:
```

Comment:
So, do we always print additional data in json mode? What if this is not verbose mode?

Reply:
according with Corin's comment, --json options should be the highest verbosity.
https://jira.hyperledger.org/browse/INDY-1443?focusedCommentId=46831&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-46831